Tasks/HelmDeployV0/src/utils.ts

```diff
@@ -70,6 +71,8 @@ export function getManifestsFromRelease(helmCli: helmcli, releaseName: string):
     helmCli.setCommand('get');
     helmCli.addArgument('manifest');
     helmCli.addArgument(releaseName);
+    if (namespace)
+        helmCli.addArgument('--namespace '.concat(namespace));
```

Comment:
Add each argument separately? Doesn't it convert to `"--namespace my-namespace"` instead?

Reply:
Yes, but that's what it should be right? This is how it's done in `helm install` and `helm upgrade` commands. Wanted to keep it consistent.